py-pkg/dummy_data/es_5_mapping.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
totally cool to check all of these in now. Can you please create an issue (you can do it before we've even merged this, honestly) indicating that we need to unify the CI strategies for R and Python?

Here's an example (if you want to link it) of testing R and Python packages in different sub-builds within one travis build using their nifty `matrix` thing: https://github.com/dmlc/xgboost/blob/master/.travis.yml#L15

Reply:
#85